x-pack/plugins/rule_registry/common/assets/field_maps/technical_rule_field_map.ts

```diff
@@ -189,6 +189,16 @@ export const technicalRuleFieldMap = {
     array: false,
     required: false,
   },
+  [Fields.SERVICE_NAME]: {
+    type: 'keyword',
+    array: false,
+    required: false,
+  },
+  [Fields.TRANSACTION_TYPE]: {
```

Comment:
These are not technical fields, they are APM-specific metadata.

Reply:
Dario, you are right! Should we just add a new file `apm_rule_field_map.ts` in this folder for now `x-pack/plugins/rule_registry/common/assets/field_maps` and we can add a new function `parseApmFields` and add it to the function [parseAlert](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/pages/alerts/components/parse_alert.ts#L36)

Or do you APM folks already owned a package where we can put these new files?

Reply:
They should be added to the `initializeIndex` function when APM rules are registered with the rule registry.

Reply:
That will be good but it might be hard for them to get that done before FF, maybe they can create an issue to fix it

Reply:
Xavier explained the context of this to me so I am ok with his proposal